### PR TITLE
Update Merge testnet example to longterm networks

### LINKED ISSUE #1088: Refactor Merge testnet tutorial
Refactor [Merge testnet tutorial](https://besu.hyperledger.org/en/stable/Tutorials/Merge-Testnet/) to be a tutorial on running a node/validator on one of the long-lived post-Merge testnets (Goerli or Sepolia).

Maybe add a Ropsten pre-Merge tutorial (@matt-nelson-csi has this content drafted - see https://hackmd.io/bF0kygj4S92fmuZNRQXuSw).

### PR DESCRIPTION
**Pull request by rolandtyler**
Signed-off-by: Roland Tyler <roland.tyler@consensys.net>

## Pull request checklist

Use the following list to make sure your PR fits the Besu documentation quality standard.

### Before creating the pull request

Make sure that:

- [x] [All commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [x] You've read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Documentation).
- [x] You've [previewed your changes locally](https://wiki.hyperledger.org/display/BESU/Preview+the+documentation).

## Describe the change
Update testnet example to Sepolia. Kiln to be deprecated.

## Issue fixed
Fixes #1088 

## Impacted parts

For content changes:

- [x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] CircleCI workflow
- [ ] Build and QA tools (for example, lint or vale)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] Read the Docs configuration
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [x] You've fixed any issues raised by the tests.
- [x] You've [previewed your changes on Read the Docs](https://wiki.hyperledger.org/display/BESU/Preview+the+documentation)
  and added a [preview link](#preview).

## Preview
https://hyperledger-besu--1108.org.readthedocs.build/en/1108/Tutorials/Merge-Testnet/

### CODE REVIEW

**Review by alexandratran — COMMENTED** [review]
I'm seeing in a few places that Sepolia is permissioned, so no one can run a validator on that network as simply as this. See https://notes.ethereum.org/zvkfSmYnT0-uxwwEegbCqg. So this tutorial should probably be for Prater/Goerli and/or Ropsten (hence the Ropsten launchpad). Or also add the caveats and instructions for requesting to become a long-term validator on Sepolia. Can you confirm this, and also make sure to test the updated tutorial?

File: docs/Tutorials/Merge-Testnet.md [hunk 1]
```diff
@@ -2,12 +2,18 @@
 Description: How to run Besu and Teku on the Merge testnet
 ---
 
-# Run Besu and Teku on the Merge testnet
+# Run Besu and Teku on a post-merge testnet
```
Comment (alexandratran) [inline docs/Tutorials/Merge-Testnet.md:5]:
```suggestion
# Run Besu and Teku on a post-Merge testnet
```

File: docs/Tutorials/Merge-Testnet.md [hunk 2]
```diff
@@ -2,12 +2,18 @@
...
 
 You can test Besu as an [execution client](../Concepts/Merge.md#execution-clients) and
 [Teku](https://docs.teku.consensys.net/en/stable/)
-as a [consensus client](../Concepts/Merge.md#consensus-clients) on the
-[Kiln Merge testnet](https://blog.ethereum.org/2022/03/14/kiln-merge-testnet/).
+as a [consensus client](../Concepts/Merge.md#consensus-clients) on a post-merge testnet.
```
Comment (alexandratran) [inline docs/Tutorials/Merge-Testnet.md:9]:
```suggestion
as a [consensus client](../Concepts/Merge.md#consensus-clients) on a post-Merge testnet.
```

File: docs/Tutorials/Merge-Testnet.md [hunk 3]
```diff
@@ -30,51 +36,73 @@ openssl rand -hex 32 | tr -d "\n" > jwtsecret.hex
...
+Besu as an execution client.
+
+## 4. Generate validator keys and stake ETH
 
 If you're running a [validator client](#beacon-node-and-validator-client), create a test Ethereum address (you can do
 this in
 [MetaMask](https://metamask.zendesk.com/hc/en-us/articles/360015289452-How-to-create-an-additional-account-in-your-wallet)).
-Fund this address with testnet ETH using the [Kiln Faucet](https://faucet.kiln.themerge.dev/).
+Fund this address with testnet ETH using a faucet.
+You can find a list of [Goerli faucets](https://github.com/eth-clients/goerli#meta-data-g%C3%B6rli) and [Sepolia faucets](https://github.com/eth-clients/sepolia#meta-data-sepolia).
```
Comment (alexandratran) [inline docs/Tutorials/Merge-Testnet.md:90]:
```suggestion
See the list of [Goerli faucets](https://github.com/eth-clients/goerli#meta-data-g%C3%B6rli) and [Sepolia faucets](https://github.com/eth-clients/sepolia#meta-data-sepolia).
```

File: docs/Tutorials/Merge-Testnet.md [hunk 4]
```diff
@@ -104,30 +146,35 @@ option.
...
+      --log-destination console                                 \
+      --rest-api-enabled=true                                   \
+      --validators-proposer-default-fee-recipient=<ETH address> \
+      --validator-keys=<path to key file>:<path to mnemonic file>[,<path to key file>:<path to mnemonic file>,...] \
+      --p2p-discovery-bootnodes "enr:-Iq4QMCTfIMXnow27baRUb35Q8iiFHSIDBJh6hQM5Axohhf4b6Kr_cOCu0htQ5WvVqKvFgY28893DHAg8gnBAXsAVqmGAX53x8JggmlkgnY0gmlwhLKAlv6Jc2VjcDI1NmsxoQK6S-Cii_KmfFdUJL2TANL3ksaKUnNXvTCv1tLwXs0QgIN1ZHCCIyk"
+    ```
+
+=== "Sepolia"
+
+    Sepolia is a permissioned network and can't be run using a validator client without [becoming a validator first](https://notes.ethereum.org/zvkfSmYnT0-uxwwEegbCqg).
```
Comment (alexandratran) [inline docs/Tutorials/Merge-Testnet.md:166]:
```suggestion
    Sepolia is a permissioned network and can't be run using a validator client without [requesting to become a validator first](https://notes.ethereum.org/zvkfSmYnT0-uxwwEegbCqg).
```

File: docs/Tutorials/Merge-Testnet.md [hunk 5]
```diff
@@ -160,4 +207,5 @@ After starting Besu and Teku, your node should start syncing and connecting to p
         2022-03-21 20:44:12.353 INFO  - Syncing     *** Target slot: 76096, Head slot: 3519, Remaining slots: 72577, Connected peers: 9
         ```
 
+You can check your validator status by searching your Ethereum address on the [Prater Beacon Chain explorer](https://prater.beaconcha.in/).
```
Comment (alexandratran) [inline docs/Tutorials/Merge-Testnet.md:210]: Can we add a note that explains that **Prater** and **Goerli** and two names for the same chain?

Comment (rolandtyler) [reply docs/Tutorials/Merge-Testnet.md]: Goerli's beacon chain is now up and running, so I just removed Prater completely

**Review by daniellehrner — APPROVED** [review]

**Pull request merged**